### DISCUSSION TITLE
[Feature] Export/Sync Album to Cloud storage

### DISCUSSION THREAD

**Discussion by NiklasRosenstein** [immich-app/immich#6090] [Feature Request]
I currently have Immich accessible over a public URL but would like to move it inside the VPN for added security. The thing that prevents me from doing that right now is that it makes sharing a set of photos for people not on the VPN much more difficult (like not-immediate family members).

I think it would be nice to have a kind of export or synchronisation feature to have Immich upload the contents of an album to cloud storage, like Google Drive or Dropbox (could be synced there continuously), or WeTransfer or even S3 (export a snapshot
of the album as ZIP file), etc.

**Comment by bo0tzz** [comment]
We're not planning to add a sync like this into Immich. What we do want to add at some point is things like calling external webhooks on Immich events, which should make it possible to build features like this external to Immich.
